Makefile

```diff
@@ -53,7 +53,14 @@ embedmd:
 	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
 	go get github.com/campoy/embedmd
 
-docs: embedmd
+apidocgen:
+	@GOOS=$(shell uname -s | tr A-Z a-z) \
+	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
```

Comment:
I reused this from the `promu` target in the Prometheus `Makefile`, not sure if it is actually required. In my tests it worked fine without, but I thought it was there for a reason in the Prometheus `Makefile`.